docs/stability.rst

```diff
@@ -252,7 +252,7 @@ The current planned and existing sub-packages are:
                 <span class="stable"></span>
             </td>
             <td>
-                New in v0.2. Adapted from <tt class="docutils literal"><span class="pre">pnbody</span></tt> and integrated into Astropy. Current functionality stable with intent to maintain backwards compatibility. Significant new functionality is likely to be added in future versions.
+                New in v0.2. Current functionality stable with intent to maintain backwards compatibility. Significant new functionality is likely to be added in future versions.
```

Comment:
I agree that with the logarithmic units present, I don't see much change coming to units; however, we do hope to have uncertainties eventually, though I don't know whether this will be in `Quantity`... @eteq - apart from that, this looks good, so feel free to merge if you think the uncertainties might end up in units.